key-script

```diff
@@ -6,7 +6,7 @@ set -e
 . /etc/ykluks.cfg
 
 if [ -z "$WELCOME_TEXT" ]; then
-    WELCOME_TEXT="Please insert yubikey and press enter or enter a valid passphrase"
+    WELCOME_TEXT="Please insert yubikey and press enter (and optionally the Yubikey button) or enter a valid passphrase"
```

Comment:
In my opinion it would not be the default behaviour to wait for a button press in case of a simple challenge response.

Thus I would like to keep the old ``WELCOME_TEXT`` and leave the additional note to be configured by the user.

Reply:
Yes that's fair, the default at least in the [UI is not to require a touch](https://github.com/Yubico/yubikey-personalization-gui/blob/63fdf3c62459575b5e7d0f7b15ebfd416fffa95f/src/ui/settingpage.cpp#L193).